src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,68 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { useState } from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  onSearchQuery: React.Dispatch<React.SetStateAction<string>>,
+  filter: string,
+  onFilter: React.Dispatch<React.SetStateAction<string>>
```

Comment:
im not sure about using dispatch

Reply:
> im not sure about using dispatch

It's ok, this a correct type of `setState` function